### DISCUSSION TITLE
Guide for API integration in flutter

### DISCUSSION THREAD

**Discussion by alishabaj** [community/community#21303] [Apps, API and Webhooks] [answered]
Dear Team, 
I am new in flutter, I have completed Basic UI, now I wanted to learn API .

**Comment by surajadkhari** [comment, marked as answer]
Doc from Flutter official
https://docs.flutter.dev/cookbook/networking/fetch-data


Also for your reference:
https://medium.com/@surajadkhari/using-future-builder-to-fetch-api-9ec48da15c8b

**Reply by alishabaj** [reply]
Thank you I will follow doc, right away
